.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
I would swap this to a husky `pre-push`. It's okay for linting to fail locally, it's only when shared with others one need to make sure it passes. Then one could drop `lint-staged` as well, since on `pre-push` its better to run full linting.

Reply:
I don't feel strong but IMHO `pre-commit` seems more common used in the community.  As in those cases, it sounds a good fit to use `git commit -n`. :)